index.html

```diff
@@ -200,53 +200,22 @@ <h1>About me</h1>
                         </ul>
                     </li>
                 </ul>
-                <!--<div class="tech">
-                    <div class="openable">
-                        <p>Languages</p>
-                        <img src="images/Techs/icons8-arrow-down-50.png">
-                    </div>
-                    <div class="techflex">
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-javascript-48.png">
-                            <p>JavaScript</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-html5-48.png">
-                            <p>HTML</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-css3-48.png">
-                            <p>CSS</p>
-                        </div>
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Frameworks</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Skills</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">-->
+
             </div>
         </section>
         <section class="comments">
             <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form_comment">
-                <input type="text" id="fname" name="fname" class="input_comment" placeholder="Full Name"><br>
+            <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
+                <input type="text" id="fname" name="fname" placeholder="Full Name" min="4" max="30" class="input_place input_comment" required>
+                <hr class="InputSeparation">
+                <br>
                 <br>
                 <br>
-                <input type="text" id="faddress" name="faddress" class="input_comment" placeholder="Email Address">
+                <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" min="10" max="30" required>
+                <hr class="InputSeparation">
+                <textarea name="comment" id="comment" cols="30" rows="10" class="text_comment" type="text" min="15" max="500" placeholder="Write some text here" required>Write me something...</textarea>
+                <button type="submit" class="msjButton">Get in touch</button>
```

Comment:
You have done a great job so far 👍. Kindly fix the submit button to be centered and with the correct font weight as in the template. This would help your project look more professional and show that you have great attention to detail 💯

Figma template:
![figmacont](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/076663ad-7acf-480c-8a45-017a4dbdaeae)

Your design:
![figbutton](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/64b038be-3823-4ff8-bdef-96b8f182b05e)

Also kindly make sure the social icons have the correct color to match the template.

Reply:
- [ ] As requested by the previous reviewer your design needs to match closely the figma template, please ensure that the lines for the name and email as shown in your image below doesn't extend to the edge of the page. see the figma design from the previous reviewers comment and make the changes accordingly so your portfolio matches completely the figma design template
- your design
![lt](https://github.com/alexansaa/Personal_Portafolio/assets/89055502/cd11a286-e2df-44df-8631-1fa64e5cd400)